src/julia/darray.jl

```diff
@@ -128,6 +161,21 @@ function foxLi{T<:BlasComplex}(::Type{T}, n::Integer, ω::Real)
 end
 foxLi(n::Integer, ω::Real) = foxLi(Complex128, n, ω)
 
+function uniform{T<:BlasFloat}(::Type{T}, m::Integer, n::Integer)
```

Comment:
I don't think we need to call Elemental to generate uniformly distributed variates. Something like `uniform{T}(::Type{T}, (m,n)) = scale!(drand(T, (m,n)) - T(0.5), 2)` should be fine, I think.